### PR TITLE
 Fix UI of dialog boxes

### PR DESCRIPTION
**Pull request by Adam-GitHub0**
In this pull request ,I changed the design of the alert dialog boxes that appears when creating a post and made them better looking and more intuitive .

### CODE REVIEW

**Review by hamzaremmal — CHANGES_REQUESTED** [review]
Can you make the background of the text invisible, so we can only have white text over blue background instead of the grey or white one.

adam

**Comment by Adam-GitHub0** [comment]
> Can you make the background of the text invisible, so we can only have white text over blue background instead of the grey or white one.
> 
> adam

Okay , I changed it in the last commit !

**Review by hamzaremmal — APPROVED** [review]

File: app/src/main/java/com/github/versus/announcements/ChoosePostSportDialogFragment.java [hunk 1]
```diff
@@ -24,47 +34,126 @@
...
+            }
+        });
+
+        return dialog;
+    }
+
+    /**
+     * Callback for when the dialog is starting.
+     */
+    @Override
```
Comment (Adam-GitHub0) [inline app/src/main/java/com/github/versus/announcements/ChoosePostSportDialogFragment.java:121]: This code is refactored in the next pull request .

File: app/src/main/java/com/github/versus/announcements/ChoosePostSportDialogFragment.java [hunk 2]
```diff
@@ -24,47 +34,126 @@
...
+    public void onStart() {
```
Comment (Adam-GitHub0) [inline app/src/main/java/com/github/versus/announcements/ChoosePostSportDialogFragment.java:122]: This method and the one above is refactored in the next P.R

**Pull request merged**